models/cow_protocol/tx_hash_labels/stable_to_stable/ethereum/cow_protocol_tx_hash_labels_stable_to_stable.sql

```diff
@@ -0,0 +1,38 @@
+{{
+    config(
+        alias='stable_to_stable_ethereum',
+        post_hook='{{ expose_spells(\'["ethereum"]\', "sector", "tx_hash_labels", \'["gentrexha"]\') }}'
+    )
+}}
+
+with
+ stable_to_stable_trades as (
+    select
+        distinct tx_hash
+    from {{ ref('cow_protocol_ethereum_trades') }}
+    where token_pair in (
+            'USDT-DAI',
+            'USDT-USDC',
+            'USDT-BUSD',
+            'DAI-USDT',
+            'DAI-USDC',
+            'DAI-BUSD',
+            'USDC-USDT',
+            'USDC-DAI',
+            'USDC-BUSD',
+            'BUSD-USDC',
+            'BUSD-USDT',
+            'BUSD-DAI'
+        )
```

Comment:
Would be cool if there was a stable_coins spell instead of this. 
Also, we should not be using this `token_pair`, but rather `token addresses` here for the following reason:

I could deploy a token with any one or two of these symbols on mainnet right now and make a trade on Cow Protocol with one of my "Fake Stable Tokens" and it would immediately be tagged as a stable trade.

In general, this token pair column should not be used for anything because we should be matching all this stuff on token addresses.

See this query and try out some others: https://dune.com/queries/1621803?TokenSymbol=BUSD 

<img width="641" alt="Screenshot 2022-11-16 at 14 36 24" src="https://user-images.githubusercontent.com/11778116/202194863-95f6f5b9-40e7-48e4-80b9-62bd217e5898.png">

Reply:
I agree and for that reason I created another PR to migrate the stablecoins table to V2 [here](https://github.com/duneanalytics/spellbook/pull/1975). Once that is implemented I'll base my labels on that.